addons/mail/static/src/core/common/discuss_model.js

```diff
@@ -0,0 +1,15 @@
+/* @odoo-module */
+
+import { toRaw } from "@odoo/owl";
+
+export class DiscussModel {
+    /** @param {DiscussModel} record */
+    eq(record) {
+        return toRaw(this) === toRaw(record);
+    }
+
+    /** @param {DiscussModel[]} collection */
+    in(collection) {
```

Comment:
collection looks like a generic word (list, map, set, ...), if it's just list, then I'd call it list (or maybe recordSet but it's misleading as we don't actually have a set)

Reply:
or just `records`

Reply:
The problem on `reactive` affects all kind of collections.
Instead of relying on `.some` that is just for arrays, we could for of loops?

Reply:
If we need to support it at some point we can always adapt it.

Reply:
I changed it to `list`